scripts/convert.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+import csv
+import json
+import os
+
+
+def csv_to_json(dir_path, filename, output_dir_path=None, labels=[]):
```

Comment:
Please only use immutable default arguments.

Take a look here:
https://docs.python-guide.org/writing/gotchas/

Reply:
Any suggestions? I've renamed it to `csv_file_to_json`.